src/test/scala/com/github/sstone/amqp/ConsumerSpec.scala

```diff
@@ -194,7 +217,17 @@ class ConsumerSpec extends ChannelSpec {
 
       // you cannot have more than 1 exclusive consumer on the same queue
       consumer1 ! AddQueue(queue)
-      val Amqp.Error(_, reason) = receiveOne(1 second)
+      val Amqp.Error(_, reason) = probe.receiveOne(1 second)
+    }
+    "save sender for requests while disconnected" in {

```

Comment:
this is a new test to make sure `ask` works if you make a request while the actor is in disconnected state.

previously, it would time out (because the reply didn't go back to the `ask`er)